api/models/Site.js

```diff
@@ -48,6 +48,12 @@ module.exports = {
       obj.siteRoot = `http://${s3Config.bucket}.s3-website-${s3Config.region}.amazonaws.com`
       obj.viewLink = obj.domain || [obj.siteRoot, 'site', obj.owner, obj.repository].join('/')
 
+      for (key in obj) {
+        if (obj[key] === null) {
+          obj[key] = undefined
+        }
+      }
```

Comment:
I'm curious: why do we need to scrub these representations and transform null values to undefined? What does it mean semantically? I presume it's related to our JSON stringification...

It's done here and with the User model.